plugin/storage/kafka/options.go

```diff
@@ -43,13 +43,17 @@ const (
 	suffixCompression      = ".compression"
 	suffixCompressionLevel = ".compression-level"
 	suffixProtocolVersion  = ".protocol-version"
+	suffixLingerMS         = ".linger-ms"
```

Comment:
time units can be defined by user, no need to fix to ms

Reply:
I was debating with myself about that.  The rational for having ms was to get it close to the official Kafka docs. It uses linger.ms in multiple places in the document.  ie. Kafka users may easily make the association from linger-ms to linger.ms.

Reply:
it's more important to be consistent internally in Jaeger. When we have timeout params, we always use time.Duration and allow user to specify the units. The help text can provide examples, like `e.g. 1s or 500ms`.

Reply:
Changed to the suggested implementation in the latest commit.